app/src/main/java/com/example/demoquizwithandroid/App.kt

```diff
@@ -0,0 +1,17 @@
+package com.example.demoquizwithandroid
+
+import android.app.Application
+import android.content.res.Resources
+
+class App : Application() {
+    override fun onCreate() {
+        super.onCreate()
+        Companion.resources = resources
+    }
+
+    companion object {
+        private var resources: Resources? = null
+        val appResources: Resources?
+            get() = resources
+    }
+}
```

Comment:
Let's remove this class; we shouldn't store a static reference to classes using contexts